tdp_vars_defaults/ranger/ranger.yml

```
@@ -7,27 +7,34 @@ ranger_release: ranger-2.0.1-TDP-0.1.0-SNAPSHOT-admin
 ranger_dist_file: "{{ ranger_release }}.tar.gz"
 ranger_usersync_release: ranger-2.0.1-TDP-0.1.0-SNAPSHOT-usersync
 ranger_usersync_dist_file: "{{ ranger_usersync_release }}.tar.gz"
+ranger_kms_release: ranger-2.0.1-TDP-0.1.0-SNAPSHOT-kms
+ranger_kms_dist_file: "{{ ranger_kms_release }}.tar.gz"
 
 # Ranger users and group
 ranger_user: ranger
+ranger_kms_user: keyadmin
 hadoop_group: hadoop
 
 # Ranger installation directory
 ranger_root_dir: /opt/tdp
 ranger_install_dir: "{{ ranger_root_dir }}/ranger"
 ranger_usersync_install_dir: "/opt/tdp/ranger-usersync"
+ranger_kms_install_dir: "/opt/tdp/ranger-kms"
```

```suggestion
ranger_usersync_install_dir: "{{ ranger_root_dir }}/ranger-usersync"
ranger_kms_install_dir: "{{ ranger_root_dir }}/ranger-kms"
```

These directories should be templated using the `ranger_root_dir` variable. Can you do it please ?

(It might not be clear why it's done for usersync in this PR, but since you're modifying adjacent variables, you can do it.)